UPGRADE.md

```diff
@@ -36,6 +73,9 @@ Following Function removed:
 
 The `TagManager` doesn't longer depend on FieldDescriptorFactory, TagEntityName and UserRepository so its constructor changed.
 
+
+The format is optional and will fallback to html if not given.
```

Comment:
Doesn't look like that part belongs here :slightly_smiling_face:

Reply:
ups did fix it